react-search-component/my-app/src/SearchBar.jsx

```diff
@@ -0,0 +1,23 @@
+import { FaSearch } from 'react-icons/fa';
+
+export default function SearchBar({ onCustomChange, input }) {
+  function handleChange(e) {
+    const { value } = e.target;
```

Comment:
This works! But no need to be fancy with destructuring! 

We can reduce this to one line: 
`onCustomChange(e.target.value);`